src/handlers/comment/handlers/payout.ts

```diff
@@ -40,3 +41,26 @@ export const payout = async (body: string) => {
   const response = await handleIssueClosed();
   return response;
 };
+
+export const autoPay = async (body: string) => {
+  const context = getBotContext();
+  const _payload = context.payload;
+  const logger = getLogger();
+
+  const payload = _payload as Payload;
+  logger.info(`Received '/autopay' command from user: ${payload.sender.login}`);
+
+  const pattern = /^\/autopay (true|false)$/;
+  const res = body.match(pattern);
+
+  if (res) {
+    const userPermission = await getUserPermission(payload.sender.login, context);
+    if (userPermission !== "admin") {
+      return "Permission required to set autopay mode.";
+    }
+    if (res) {
```

Comment:
What does this line do? You are already checking like `if (res)`. Maybe it should be like `if (res.length > 1)`?!?!

Reply:
fixed